docs/models.yml

```diff
@@ -258,7 +264,7 @@ committee:
 meeting:
   id: number
   welcome_title: string
-  welcome_text: HTMLVideo
+  welcome_text: HTMLStrictPermissive
```

Comment:
This is wrong.

Reply:
ops, dont know how that happened. my bad